tests/schema/test.formats.js

```diff
@@ -41,6 +41,8 @@ describe('formats', () => {
       '1.0.0-rc1.0+001',
       '0.1.12dev-cb31c51',
       '4.1.1dev-abcdef1',
+      // eslint-disable-next-line prefer-template
+      '1.9' + '9'.repeat(99),
```

Comment:
You can have expressions in a template string so this could be:

```javascript
`1.${'9'.repeat(100)}`
```
and save on the eslint override.